.travis.sh

```diff
@@ -86,6 +86,8 @@ case "$JDK" in
   ;;
 9-ea | 9-ea-stable)
   # see https://bugs.openjdk.java.net/browse/JDK-8131041 about "java.locale.providers"
```

Comment:
In fact will be better to get rid of this `java.locale.providers` completely, but that's another story and I'll try to handle it separately.